### DISCUSSION TITLE
http://0.0.0.0:4000 on Windows Docker

### DISCUSSION THREAD

**Discussion by adaliszk** [jekyll/jekyll#8602] [Q&A]
Hey,

I was trying to use Docker on Windows to run Jekyll, I am using the github-pages package to install Jekyll, so it's a bit behind in versions with 3.9.0. None the less, I set the `JEKYLL_ENV` to be `docker` and load config files with `url: "http://localhost:4000"`. The docker itself launches Jekyll with `--host 0.0.0.0` and that's being reported nicely.

The page loads, at a first glance everything looks okay, but any link to other pages becomes: `http://0.0.0.0:4000` which would work on Linux or Mac, but it's not allowed on Windows. The theme I'm using uses the `relative_url` directive, so in theory, it should not even have any absolute links at all to itself, yet it does.

Does anybody have an idea what could I do?

**Comment by Michaelcorpin** [comment]
how can i retrieve my facebook account. it has been hacked by ***@***.***
